src/server/cph-number/controller.js

```diff
@@ -0,0 +1,68 @@
+export const pageTitle =
+  'What is the County Parish Holding (CPH) number of your farm or premises where the animals are moving off?'
+
+/**
+ * The first question to start the journey.
+ * @satisfies {Partial<ServerRoute>}
+ */
+export const getController = {
+  handler(req, h) {
+    const cphNumber = req.yar.get('cphNumber')
+
+    return h.view('cph-number/index', {
+      pageTitle,
+      heading: pageTitle,
+      cphNumber: {
+        value: cphNumber
+      }
+    })
+  }
+}
+
+/**
+ * Respond to the first question.
+ * @satisfies {Partial<ServerRoute>}
+ * @param req
+ */
+export const postController = {
+  handler(req, res) {
+    const valid = /([0-9]{2})\/([0-9]{3})\/([0-9]{4})/g
```

Comment:
Not something to do now, but something to think about for the future: 

We're likely to have to re-validate at least some data on the declaration screen - e.g. when users submit data that depends on answer to previous questions.

There's nothing stopping a user from updating a single question (and invalidating some previously submitted state - such as updating the 'earliest' and 'latest' dates, making the latest date before the earliest).

When they do that, we'll have to detect what they'e made invalid so that we can guide them through those pages *and* prevent them from declaring and submitting if they deep link straight back to the summary page. 

Also, we have access to `joi`, so we might find there's a more direct way of embedding error messages in there.